src/pudl/transform/ferc1.py

```diff
@@ -1033,10 +1119,10 @@ def merge_instant_and_duration_tables_xbrl(
 
         if instant.empty:
             logger.info(f"{self.table_id.value}: No XBRL instant table found.")
-            return duration
+            out_df = duration
```

Comment:
i removed these multi returns and instead  had all options generate an `out_df` that gets returned with one return